tests/ci/test-vm.sh

```diff
@@ -81,7 +81,7 @@ test_gnome_is_running () {
 test_neptune_is_running () {
     echo "Checking if Neptune is running"
 
-    assert_process_running "neptune3-ui"
+    assert_process_running "/usr/lib64/neptune3/neptune3-ui"
```

Comment:
I had to put the full path for the process, because there are a few processes that contains `neptune3-ui` and the assert check if there is 1 occurrence.